// @polkadot/util/bi/index.cjs
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
var _exportNames = {
  biSqrt: true,
  biToBigInt: true,
  biMax: true,
  biMin: true,
  biToHex: true,
  biToU8a: true
};
Object.defineProperty(exports, "biMax", {
  enumerable: true,
  get: function () {
    return _max.biMax;
  }
});
Object.defineProperty(exports, "biMin", {
  enumerable: true,
  get: function () {
    return _min.biMin;
  }
});
Object.defineProperty(exports, "biSqrt", {
  enumerable: true,
  get: function () {
    return _sqrt.biSqrt;
  }
});
Object.defineProperty(exports, "biToBigInt", {
  enumerable: true,
  get: function () {
    return _toBigInt.biToBigInt;
  }
});
Object.defineProperty(exports, "biToHex", {
  enumerable: true,
  get: function () {
    return _toHex.biToHex;
  }
});
Object.defineProperty(exports, "biToU8a", {
  enumerable: true,
  get: function () {
    return _toU8a.biToU8a;
  }
});

var _consts = require("./consts.cjs");

Object.keys(_consts).forEach(function (key) {
  if (key === "default" || key === "__esModule") return;
  if (Object.prototype.hasOwnProperty.call(_exportNames, key)) return;
  if (key in exports && exports[key] === _consts[key]) return;
  Object.defineProperty(exports, key, {
    enumerable: true,
    get: function () {
      return _consts[key];
    }
  });
});

var _sqrt = require("./sqrt.cjs");

var _toBigInt = require("./toBigInt.cjs");

var _max = require("./max.cjs");

var _min = require("./min.cjs");

var _toHex = require("./toHex.cjs");

var _toU8a = require("./toU8a.cjs");
